dbus/src/arg/mod.rs

```diff
@@ -405,6 +414,7 @@ pub enum ArgType {
     /// f64
     Double = ffi::DBUS_TYPE_DOUBLE as u8,
     /// File
+    #[cfg(unix)]
     UnixFd = ffi::DBUS_TYPE_UNIX_FD as u8,
```

Comment:
So I don't think we should remove ArgType or MessageItem variants. (Btw, there are transports on unix as well where fd passing is not supported.) If we're on a system where fd passing is not supported then people would just not send or receive that argtype or messageitem.

Where would we run into issues if we kept them?

Reply:
Removed `#[cfg(unix)]` from `ArgType::UnixFd` and from the corresponding entry in `ALL_ARG_TYPES` - no issues

Reply:
FromRawFd, ToRawFd, RawFd are unix-only. Any reference to them won't compile on Windows.

1. When I add enum variants back, I get `non-exhaustive patterns: 'UnixFd' not covered`
2. When I remove `#[cfg(unix)]` there - necessary traits not implemented (the trait `msgarg::Get<'_>` is not implemented for `File`, the trait `msgarg::RefArg` is not implemented for `File`)
3. When I remove `#[cfg(unix)]` from trait implementations - From/To/RawFd unavailable (`impl RefArg for File` needs `self.as_raw_fd()`)

It should be possible to keep UnixFd enum variants, if these trait implementations will just always panic on Windows

Reply:
Added as a separate commit